### PR TITLE
Rename search app to admin

### LINKED ISSUE #805: Rename the search app to admin
The search app won't be just search forever, it will eventually be the entire admin. The AMO pages will also get search so it will just be confusing.

### PR DESCRIPTION
**Pull request by tofumatt**
After chatting with @muffinresearch I understand this a bit better 😄

Fixes #805.

r?

### CODE REVIEW

**Comment by muffinresearch** [comment]
As mentioned before ops will need to update their deploy target. Probably worth talking to them before you land it.

lgtm r+wc (definitely worth getting mstriemer to have a look since he is more familiar with the app)

**Comment by tofumatt** [comment]
Will do on both front, thanks :-)

File: package.json [hunk 1]
```diff
@@ -156,14 +156,15 @@
     "piping": "0.3.2",
     "react": "15.3.1",
     "react-addons-css-transition-group": "15.3.1",
+    "redux-async-connect": "1.0.0-rc4",
```
Comment (mstriemer) [inline package.json]: Did you mean to add these dependency changes?

**Comment by mstriemer** [comment]
r+wc

File: package.json (hunk 1 above)
Comment (tofumatt) [inline package.json]: Whoops, no, it was from me tweaking some `npm` version oddness. I'll remove 'em.

**Comment by tofumatt** [comment]
Pinging @jasonthomas before this lands--we'll need to update the deploy target for the admin service to be "admin" instead of "search". I guess that'd need to handle on DEV before I merge and then happen during next week's push of stage/prod.

File: package.json (hunk 1 above)
Comment (tofumatt) [inline package.json]: Though in this line's case it's just alphabetising. I don't recall doing that manually, but I'll make sure it's not changing any versions and push a fix.

**Comment by jasonthomas** [comment]
I've made those changes in https://github.com/mozilla-services/puppet-config/pull/2234

**Comment by tofumatt** [comment]
Cool, I'll need to rebase this against master. I'll aim to get it done this week 😊

**Comment by mstriemer** [comment]
I hit merge on my PR and then immediately remembered about this 😞. I can resolve the conflicts and merge it if you'd like.

**Comment by mstriemer** [comment]
(I think you said you had PTO coming up)

**Comment by tofumatt** [comment]
No worries! One of us would've had to resolve and I think my PR is simpler
conceptually.

I'm only PTO tomorrow as I'm driving from England back to Edinburgh and
getting a motorbike. I should be able to get to it tomorrow evening or
Friday at the latest. I'll do it 😊

- tofumatt

**Pull request merged**